src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { Comment } from '../../types/comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[],
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <>
+    <div className="CommentList">
+      {comments.map(comment => (<CommentInfo comment={comment} />))}
```

Comment:
You need to add the key attribute to the list elements, think about how to do it

Reply:
fixed